Source/plrmsg.cpp

```diff
@@ -25,16 +26,27 @@ namespace {
 
 uint8_t plr_msg_slot;
 _plrmsg plr_msgs[PMSG_COUNT];
+int msgs;
 
 /** Maps from player_num to text color, as used in chat messages. */
 const UiFlags TextColorFromPlayerId[MAX_PLRS + 1] = { UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhite, UiFlags::ColorWhitegold };
 
+int GetChatLines(int width, char *textPtr)
+{
+	const size_t length = strlen(textPtr);
+	const string_view text { textPtr, length };
+	std::string lineCountString = WordWrapString(text, width);

```

Comment:
It looks like this might only be calculating the number of lines needed for the message, but what is actually being printed is the player name + message, so the two might not align.